include/cantera/kinetics/RxnRates.h

```diff
@@ -439,11 +459,11 @@ class Plog
  * therefore extrapolation of rates outside the range of temperatures and
  * pressures for which they are defined is strongly discouraged.
  */
-class ChebyshevRate
+class Chebyshev
```

Comment:
I was trying to remove the suffix `Rate` as each of the original rate expressions in `RxnRates.h` is incorporated into a new object in `ReactionRate.h`, which ends in `Rate`. Once CTI/XML are gone, it may be possible to merge the two, and I didn't see an immediate need to introduce `Chebyshev` elsewhere.